.circleci/deployment/workflows.yml

```diff
@@ -37,21 +37,21 @@
           filters:
             branches:
               only:
-                - develop
+                - 1544-Nginx-with-domain # switch back to develop before merge
       - deploy-develop:
           requires:
             - deploy-infrastructure-develop
           filters:
             branches:
               only:
-                - develop
-      - test-deployment-e2e:
-          requires:
-            - deploy-develop
-          filters:
-            branches:
-              only:
-                - develop
+                - 1544-Nginx-with-domain # switch back to develop before merge
+      #- test-deployment-e2e:
```

Comment:
Need to investigate changing up this line:

tdrs-frontend/package.json
```
    "test:e2e-develop": "cypress run --headless -b chrome --config baseUrl=https://${BASE_URL_PREFIX}-develop.app.cloud.gov --env cypressToken=${CYPRESS_TOKEN},apiUrl=https://${API_URL_PREFIX}-develop.app.cloud.gov/v1",
```

Reply:
@raftmsohani this should be switched back I think. It will fail until we re-update the variables in cloud.gov